components/molecule/notification/src/index.scss

```diff
@@ -26,7 +25,11 @@
   }
 
   &-close {
+    @include media-breakpoint-down(xSmall) {
```

Comment:
we decided to not allow `media-breakpoint-down` anymore and only use `media-breakpoint-up` instead

https://confluence.schibsted.io/pages/viewpage.action?spaceKey=ST&title=Breakpoints+-+Frontend